backend/main.py

```
@@ -141,6 +141,19 @@ def get_recent_audit_logs(self, limit): return []
 # Initialize AI Firm
 AI_FIRM_READY = False
 RL_ENV_READY = False
+
+
+class MockDebateEngine:
+    async def conduct_debate(self, symbol, context):
```

Ruff (0.15.5)

[warning] Unused method argument: `context`

(ARG002)